### PR TITLE
feat: add visual distinction in data browser for internal classes and display their real names with underscore

### PR DESCRIPTION
**Pull request by damianstasik**
### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).

### Issue Description
This PR refactors parts of the app that shows class names so that it displays the real name (the one with an underscore). Additionally, there is now also a separator between internal and custom classes in the browser sidebar.

Related issue: #1849

### Approach
- Show real class names
- Add separator between internal and custom classes

### TODOs before merging

- [x] A changelog entry is created automatically using the pull request title (do not manually add a changelog entry)

### CODE REVIEW

**Comment by damianstasik** [comment]
There are a few ways of implementing the separator:
1. Using separate `<CategoryList />` in the `Browser.react.js`, but that would mean the separator would be a feature only available in the browser sidebar.
2. Adding a new type of category item with a new property called `type`, in this case, it would include only `separator` and would be handled for any `CategoryList` instance.
3. Combination of 1&2 would be allowing a `component` property in the category list which could be used in `Browser.react.js` to add a custom separator only for the browser sidebar.

I chose the second option as it does not produce a duplicated code if other sections of the dashboard would also want a separator between categories.

File: src/lib/Constants.js [hunk 1]
```diff
@@ -45,14 +45,14 @@ export const Directions = {
...
 
-export const SpecialClasses = {
-  _User: 'User',
-  _Installation: 'Installation',
-  _Role: 'Role',
-  _Product: 'Product',
-  _Session: 'Session',
-  _PushStatus: 'PushStatus',
-};
+export const SpecialClasses = [
```
Comment (damianstasik) [inline src/lib/Constants.js:48]: Changed from the map into an array as the object values are no longer used anywhere.

Comment (mtrezza) [reply src/lib/Constants.js:48]: Looks fine

**Review by mtrezza — APPROVED** [review]
Looks good!

**Comment by parseplatformorg** [comment]
🎉 This pull request has been released in version [3.3.0-alpha.9](https://github.com/parse-community/parse-dashboard/releases/tag/3.3.0-alpha.9)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [3.3.0-beta.1](https://github.com/parse-community/parse-dashboard/releases/tag/3.3.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [3.3.0](https://github.com/parse-community/parse-dashboard/releases/tag/3.3.0)

**Pull request merged**